src/components/admin/tabs/add-tab.tsx

```
@@ -845,33 +647,20 @@
-            />
-            <button
-              type="button"
-              onClick={() => fileInputRef.current?.click()}
-              className="mt-2 px-4 py-2 bg-gray-200 text-gray-700 rounded-lg hover:bg-gray-300"
-            >
-              ファイルを選択
-            </button>
+          <label className="block text-gray-700 text-sm font-medium mb-1">
+            画像（登録時に自動取得）
+          </label>
+          <div className="border-2 border-dashed border-gray-300 rounded-lg p-6 text-center bg-gray-50">
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)